dependencies.yaml

```
@@ -661,6 +661,18 @@ dependencies:
         packages:
           - dask-cuda==24.4.*
           - *numba
+      - output_types: conda
+        packages:
+          - pip
+          - pip:
+              # This should eventually move to rapids-dask-dependency
+              - &dask_expr_tip git+https://github.com/dask-contrib/dask-expr.git@b588a9e15e90e0567061664ffc01374786686e20
+      - output_types: requirements
+        packages:
+          - *dask_expr_tip
+      - output_types: pyproject
+        packages:
+          - dask-expr@git+https://github.com/dask-contrib/dask-expr.git@b588a9e15e90e0567061664ffc01374786686e20
```

It would be nice to use `dask_expr_tip` here to cut down on duplication, but have no idea if something like

```suggestion
          - dask-expr @ *dask_expr_tip
```

Would work here as I'm not the most familiar with `rapids-dependency-file-generator`

---

Sounds fine to me. As you know, I'm just deferring to your advice for these kinds of changes :)

Another background note: I'm expecting dask-expr to be copied entirely into dask/dask proper before the "query-planning" default is changed. When that happens, we should be able to remove all this stuff.

---

Hmm - I'm not having much luck with this. Not sure if we _can_ avoid the duplication in this case.